src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, sex, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+
+      {age && <p className="Person__age">I am {age}</p>}
+
+      <p className="Person__partner">
+        {
+          isMarried
```

Comment:
You could calculate this and assign to const in the body of the component,
```
const personPartnerText = isMarried  ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'` : 'I am not married'
```
then just display here:
```
{personPartnerText}
```